components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -578,15 +650,17 @@ export function RepositoryNotFound(p: { error: StartWorkspaceError }) {
         .toString();
 
     if (!userScopes.includes(missingScope)) {
-        return renderError(
-            "The repository may be private. Please authorize Gitpod to access private repositories.",
-            "Grant access",
-            authorizeURL,
+        return (
+            <RepositoryInputError
+                title="The repository may be private. Please authorize Gitpod to access private repositories."
+                linkText="Grant access"
+                linkHref={authorizeURL}
+            />
```

Comment:
**suggestion**: In this case, if we used the new alert design, new disabled button design, also placed at the bottom, here's how this could look like. What do you think?

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="New" src="https://github.com/gitpod-io/gitpod/assets/120486/3a5d20bb-0bf0-4430-afdf-8cc7b226f581"> | <img width="1440" alt="New-1" src="https://github.com/gitpod-io/gitpod/assets/120486/24e869a6-e059-437a-8805-d987bb337af5"> |

Reply:
This pattern can probably catch and improve most, if not all, cases like the ones described in UX-36, WEB-361, UX-34, WEB-405, WEB-361, etc. Cc @axonasif @geropl @mbrevoort

Reply:
I like the solid alert - and I think the top feels a little better than the bottom for it's placement as it doesn't appear in the middle of the form. Some of the types of messaging we show here work pretty well with your changes.

| yaml error | permission error |
|--------|--------|
| <img width="542" alt="image" src="https://github.com/gitpod-io/gitpod/assets/367275/fe71dfbb-41ae-45e5-a67f-c0f5848d7196">
 | <img width="527" alt="image" src="https://github.com/gitpod-io/gitpod/assets/367275/9272fa95-ce2e-48eb-af6c-6bd9f74c6dcd"> | 

The `Authorize w/ github.com` link feels a little weird to me, maybe it's the blue color on that yellow? I made it semi-bold and it helped a little. I can also just go with existing alert styles, but make these other changes, and we can adjust alert styles in a followup to get them right.